src/html/chat.html

```diff
@@ -33,7 +33,40 @@
 {% endblock %}
 
 {% block content %}
-<div id="chat-wrapper">
-    <iframe id="chat-frame" src="https://kiwiirc.com/client/irc.esper.net:+6697/?nick=sponge|?#sponge"></iframe>
+<header>
+    <div class="container">
+        <div class="logo">
+            <img src="assets/img/icons/spongie-mark-reverse-dark.svg" alt="">
+            <h1>Sponge</h1>
+        </div>
+        <h2>Chat</h2>
+    </div>
+</header>
+
+<div class="container chat-content">
+    <section id="chat-overview">
+        <p>
+            SpongePowered provides several IRC channels and a Discord server for you to discuss the Sponge project on.
+            The IRC channels and Discord server are linked, so feel free to use either.
+        </p>
+    </section>
+
+    <section id="chat-irc">
+        <h3>IRC</h3>
+        <p>
+            The official Sponge channels are <code>#sponge</code>, <code>#spongedev</code>, and <code>#spongedocs</code> on <a href="irc://irc.esper.net:6667">irc.esper.net</a>.
+        </p>
+
+        <p>
+            If you don't have an IRC client installed, you can <a href="https://kiwiirc.com/client/irc.esper.net:+6697/?nick=sponge|?#sponge" target="_blank">use the web client</a>.
+        </p>
+    </section>
+
+    <section id="chat-discord">
+        <h3>Discord</h3>
+        <p>
+            <a href="https://discord.gg/PtaGRAs">Click here</a> to join the SpongePowered Discord server.
```

Comment:
Perhaps use the widget that Discord provides here?

Reply:
I agree with @kashike. I would prefer a dedicated widget if it was mostly focused on the join link (kind of like Apple's "Available on the App Store" image), but the existing widget is mainly a list of people who are on the server, with a relatively small "Connect" button (copy the link in the HTML code into a browser tab and make the window fairly small to see what it would look like).

Reply:
is there a logo we can hyperlink to use for this?

Reply:
I personally made my own little widget which just loads the current online user count, maybe you could reuse that? source is here: https://gist.github.com/phit/ba2314f6e1e6ae30ec8a57b9eb5036ca

you can see an example here: https://www.stonebound.net/assets/discord/

Reply:
I'm not sure if we really need to display the current user counts. Placing one of the Discord logos here might help to make things more clear, though.

Reply:
Found this as a download under the [StreamKit page](https://discordapp.com/streamkit):

![discord_join_blurple](https://user-images.githubusercontent.com/12983479/33231742-e949ca00-d1bf-11e7-9f87-69e00a0dafe7.png)

I'm just not sure about the "me" wording, since Sponge isn't just one person.

Reply:
I think a simple Discord logo, without any additional text would already be enough. They are available in various forms at https://discordapp.com/branding

Reply:
@Minecrell Here's the screenshot of that:
![untitled](https://user-images.githubusercontent.com/12983479/33801061-7d543672-dd14-11e7-84b1-e418a70bc988.png)
(Continues with the footer below.)

Reply:
That looks fine. Can you also add the link to the Discord invite to the image?

Reply:
Discord logo feels out of place to be honest.